test/test_data/default_test_values.py

```diff
@@ -1,3 +1,4 @@
 DEFAULT_BUSINESS_UUID = '3b136c4b-7a14-4904-9e01-13364dd7b972'
 DEFAULT_SURVEY_UUID = 'cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87'
+ALTERNATE_SURVEY_UUID1 = '110711c3-0ac8-41d3-ae0e-567e5ea1ef87'
```

Comment:
Is there already a ALTERNATE_SURVEY_UUID?

Reply:
Nope , but thee was a 2 and I forgot to rename this when I backed it out, good catch